docs/content/concepts/ops-jobs-graphs/graphs.mdx

```diff
@@ -369,7 +364,15 @@ def define_dep_dsl_graph() -> GraphDefinition:
 
 ---
 
-## Building jobs
+## Using graphs
+
```

Comment:
Can you add a list of subsection links here to make it consistent with other sections? See the "Creating graphs" section for an example.

Reply:
sure thing